src/main/java/domain/CarDTO.java

```diff
@@ -0,0 +1,10 @@
+package domain;
+
+public class CarDTO {
+    public final  String name;
+    public final int position;
+    public CarDTO(String name, int position) {
+        this.name = name;
+        this.position = position;
+    }
```

Comment:
```suggestion
    public final String name;
    public final int position;
    
    public CarDTO(String name, int position) {
```
공백이 한칸 더 들어갔네요ㅎㅎ
가독성을 위해 인스턴스 변수와 생성자 사이에 라인을 띄워주는 습관을 들여놓는 것도 좋습니다

Reply:
이번 미션에서 컨벤션을 지키지 못한 것들이 좀 있네요.😥
잘 작성하도록 하겠습니다!